network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/AbstractBranchImpl.java

```diff
@@ -375,9 +387,12 @@ public Optional<OperationalLimitsGroup> getSelectedOperationalLimitsGroup1() {
     @Override
     public OperationalLimitsGroup newOperationalLimitsGroup1(String id) {
         var resource = getResource();
-        var group = OperationalLimitsGroupAttributes.builder().id(id).build();
-        resource.getAttributes().getOperationalLimitsGroups1().put(id, group);
-        return new OperationalLimitsGroupImpl<>(this, TwoSides.ONE, group);
+        var newGroup = OperationalLimitsGroupAttributes.builder().id(id).build();
+        index.loadOperationalLimitsGroupAttributesForBranchSide(ResourceType.convert(getType()), getId(), 1);
+        OperationalLimitsGroupAttributes oldValue = resource.getAttributes().getOperationalLimitsGroups1().get(id); // can be null
+        updateResource(res -> resource.getAttributes().getOperationalLimitsGroups1().put(id, newGroup),
+                "operationalLimitsGroup1", oldValue, newGroup);
```

Comment:
in powsybl core they send notifications for each limit type: https://github.com/powsybl/powsybl-core/blob/cea16b70fbca5ca7589cccdc0dec86f3d560d2dd/iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/OperationalLimitsGroupsImpl.java#L154

Since the new value is always null there and there is a check new != old, it doesn't notify when adding a new group. To investigate, what behavior do we want?